gradle.properties

```diff
@@ -1,4 +1,14 @@
 
+org.gradle.jvmargs=-Xmx2024m -XX:MaxPermSize=512m
```

Comment:
Why does this have so many repeating lines?

Reply:
its a bug in the analyzer that only manifests itself when the analyzer runs multiple times on the same repo in the same run

Reply:
basically only happens locally coz the analysis runs multiple times on the same repo.

Reply:
Local analysis?

Reply:
it doesn't cause trouble on dev or prod tho.

Reply:
yes, local analysis

Reply:
Cool